README.md

```diff
@@ -24,6 +24,7 @@ Features
 
 - [x] [NodeJS](https://nodejs.org/en/download/) 10 or higher
 - [x] npm package manager
+- Windows 7 is only supported on version `4.2.12` and below
```

Comment:
So it works with the current version?

Reply:
I have not tested Windows 10, but I know it works on Windows Server 2012 R2.